README.md

```diff
@@ -1,67 +1,78 @@
-# Aurae
+<!-- THE DOCUMENT -->
 
-[![Main Build (lint, compile, test)](https://github.com/aurae-runtime/aurae/actions/workflows/001-cargo-install-ubuntu-make-build.yml/badge.svg?branch=main)](https://github.com/aurae-runtime/aurae/actions/workflows/001-cargo-install-ubuntu-make-build.yml) [![Documentation](https://github.com/aurae-runtime/aurae/actions/workflows/036-check-website-documentation-aurae-builder-make-check-docs.yml/badge.svg)](https://github.com/aurae-runtime/aurae/actions/workflows/036-check-website-documentation-aurae-builder-make-check-docs.yml) [![Deploy (aurae.io)](https://github.com/aurae-runtime/aurae/actions/workflows/091-deploy-website-documentation-ubuntu-make-docs.yml/badge.svg?branch=main)](https://github.com/aurae-runtime/aurae/actions/workflows/091-deploy-website-documentation-ubuntu-make-docs.yml)
+![Deploy (aurae.io)] ![Documentation] ![Main Build (lint, compile, test)]
 
-Aurae is a free and open source Rust project which houses a memory-safe systems runtime daemon built specifically for enterprise distributed systems called `auraed`.
+<p align="center">
+  <img src="https://raw.githubusercontent.com/aurae-runtime/aurae/main/docs/assets/logo-1024x1024.png" width="450">
+</p>
 
-The `auraed` daemon can be run as a PID 1 on a Linux kernel and manages containers, virtual machines, and spawning short-lived nested virtual instances of itself for an additional layer of isolation.
+# Summary
 
-### Mission
+[Aurae] is a memory-safe runtime daemon, process manager, and PID-1 initialization
+system designed to remotely schedule processes, containers, and virtual machines
+as well as set node configurations like networking storage.
 
-Aurae is on a mission to be the most loved and effective way of managing workloads on a single piece of hardware. Our hope is that by bringing a better set of controls to a node, we can unlock brilliant higher order distributed systems in the future.
 
-Aurae takes ownership of all runtime processes on a single piece of hardware, and provides mTLS encrypted gRPC APIs ([Aurae Standard Library](https://aurae.io/stdlib/)) to manage the processes. With Aurae [Cells](https://aurae.io/blog/24-10-2022-aurae-cells/) the project offers a way to slice up a system using various isolation strategies for enterprise workloads.
+The [open-source] project adapts to the expanding demands of enterprise
+distributed systems and meets the present and future demands for brilliant
+higher-order distributed systems using the [Rust] systems language.
```

Comment:
This makes it seem like higher-order distributed systems would also need to be written in rust.

Also, while "brilliant" worked with the original wording, I don't think it does anymore.

Reply:
Thank you for taking the time I highly valued your intput.

Will change it accordingly.

Reply:
> This makes it seem like higher-order distributed systems would also need to be written in rust.
> 
> Also, while "brilliant" worked with the original wording, I don't think it does anymore.

Adapted and applied.